app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+    if len(split_command) == 3:
+        if (split_command[0] == "cp"
+                and split_command[1] != split_command[2]):
+            with (open(split_command[1], "r") as file_in,
+                  open(split_command[2], "w") as file_out):
```

Comment:
Instead of retrieving different elements after split by indexing, you could unpack `split_command` and write arguments to variables thus making your code more descriptive. Change your approach to use unpacking.